schema.sql

```diff
@@ -1,5 +1,13 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id              INT GENERATED ALWAYS AS IDENTITY,
+    name            VARCHAR(150),
+    date_of_birth   DATE,
+    escape_attempts INT,
+    neutered        BOOLEAN,
+    weight_kg       DECIMAL(5,2),
+    PRIMARY KEY(id)
```

Comment:
- [ ] Good job making the `id` a primary key.

Reply:
Thank you!